lib/biran/configurinator.rb

```diff
@@ -53,7 +53,7 @@ def build_app_config
       }
 
       app_config.deep_merge! app_config_defaults
-      app_config[:secrets] = get_secret_contents(app_config)
+      app_config[:secrets].deep_merge! get_secret_contents(app_config)
```

Comment:
This follows established pattern and is backwards compatible. If no secrets block in app_config.yml then no changes will show. Only if you start adding a secrets block in app_config.yml will you see new behavior that is consistent with current behavior where the extra files add to and possible over write the values in app config.
So the order of application would be app_config.yml --> secrets.yml --> local_config.yml